### PR TITLE
fix(themes): Fix Neobrutalism theme in Core 2

### PR DESCRIPTION
**Pull request by EmmanouelaPothitou**
## Description
Fix Neobrutalism theme in Core 2

image
image

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Comment by desiprisg** [comment]
!preview

File: .changeset/smooth-pugs-deny.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/themes": patch
+---
+
+Fix Neobrutalism theme in Core 2
```
Comment (nikosdouvlis) [inline .changeset/smooth-pugs-deny.md:5]:
```suggestion
Update the `neobrutalism` to make it compatible with the new Core 2 components
```

**Review by nikosdouvlis — APPROVED** [review]

**Comment by clerk-cookie** [comment]
Hey @desiprisg, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-irrk3sy4r.clerkstage.dev) | May 02, 2024 10:49 AM |

**Comment by nikosdouvlis** [comment]
!snapshot

**Comment by desiprisg** [comment]
I'm noticing that the phone input is having some issues. Do you think we can align this as well somehow? I think that `polishedAppaearance.ts` implementation will help for this.
image

**Comment by clerk-cookie** [comment]
Hey @nikosdouvlis - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| gatsby-plugin-clerk | 5.0.0-beta.45 |
| @clerk/themes | 2.1.2-snapshot.vf47a4dc |

Tip: Use the snippet copy button below to quickly install the required packages.
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.0-beta.45 --save-exact
```
`@clerk/themes`
```sh
npm i @clerk/themes@2.1.2-snapshot.vf47a4dc --save-exact
```

**Comment by desiprisg** [comment]
Additionally, this button seems to not follow the others, and the badges seem a bit off. Not too worried about the badges but I think that the button is probably some kind of misconfiguration in our components and not the theme. Also noticing that table borders don't match as well. Added some screenshots.
image
image
image

**Review by nikosdouvlis — APPROVED** [review]

**Pull request merged**